applications/minotari_app_grpc/Cargo.toml

```diff
@@ -26,6 +26,7 @@ rand = "0.8"
 thiserror = "1"
 tonic = "0.6.2"
 zeroize = "1"
+subtle = { version = "2.5.0", features = ["core_hint_black_box"] }
```

Comment:
Good call using this feature, since it adds an additional layer of optimization protection!